dspace-api/src/main/java/org/dspace/administer/StructBuilder.java

```diff
@@ -142,13 +171,52 @@ public static void main(String[] argv)
         // set the context
         context.setCurrentUser(ePersonService.findByEmail(context, eperson));
 
+        // Export? Import?
+        if (line.hasOption('x')) {
+            exportStructure(context, output);
```

Comment:
The user is logged in by line 172.  But it is possible that 'String eperson' contains a name that matches no EPerson, and in that case the context will be logged out.  We could find the EPerson during command analysis, and exit with an error if -e was given but its value is not a known account.  Probably that should be a separate issue.  The code doesn't do anything *wrong*, but the resulting error message is less helpful than it might be.  We ought to check for this problem throughout the CLI.

Reply:
Oh, I see:  you perhaps mean that export should *require* -e as import does.  That also can be pushed up to command analysis, so that execution can assume that it has a valid identity to set into the Context.